.travis.yml

```diff
@@ -189,6 +206,45 @@ matrix:
         - docker run -d --network host --name symbolicator us.gcr.io/sentryio/symbolicator:latest run
         - docker ps -a
 
+    - <<: *postgres_default
+      name: 'Sentry-Relay integration tests'
+      env: TEST_SUITE=relay-integration DB=postgres
+      services:
+        - docker
+        - memcached
+        - redis-server
+        - postgresql
+      before_install:
+        - *before_install_default
+        - docker pull us.gcr.io/sentryio/relay:latest # pull relay we'll run and kill it for each test
+
+        - > # clickhouse
```

Comment:
_nit_: Can we merge this with the configuration used for the Snuba integration tests? It would be a nice mid-term goal to merge the two jobs into one "Integration tests" suite, later down the road.